src/api/app/jobs/project/update_packages_if_dirty.rb

```diff
@@ -0,0 +1,7 @@
+class Project
```

Comment:
> Nested where? what is the namespace for it?

Its a classes nested inside another class. A ruby class can either be nested inside a module or a class.

> What if I come after you and I add something like this in this file:
> Then I'm overriding the **name** method of the Project model

Then don't do that?